conf/params.yaml

```
@@ -31,4 +31,7 @@ atos:
       quality_of_service: ""
   trajectorylet_streamer:
     ros__parameters:
-      chunk_duration: 2.0
\ No newline at end of file
+      chunk_duration: 2.0
+  pointcloud_publisher:
+    ros__parameters:
+      pointcloud_file: [""]
```

```suggestion
      pointcloud_files: [""]
```
... with corresponding changes in code